src/renderer/components/commands-action-button.tsx

```diff
@@ -105,6 +106,26 @@ export class GistActionButton extends React.Component<
     }
   }
 
+  public async getFiddleDescriptionFromUser(): Promise<string> {
+    const { appState } = this.props;
+
+    // Reset potentially non-null last description.
+    appState.genericDialogLastInput = null;
+
+    appState.setGenericDialogOptions({
+      type: GenericDialogType.confirm,
+      label: 'Please provide a brief description for your Fiddle Gist',
+      cancel: 'Skip',
+      wantsInput: true,
+      placeholder: 'Electron Fiddle Gist',
+    });
+
+    appState.isGenericDialogShowing = true;
+    await when(() => !appState.isGenericDialogShowing);
+
+    return appState.genericDialogLastInput ?? 'Electron Fiddle Gist';
```

Comment:
The Okay button here does the same thing as the Skip one. 

If we want a true skip, this should also short-circuit to `Electron Fiddle Gist` if  `appState.genericDialogLastResult` is false.

However, I'd personally be okay with not having a `Skip` button altogether, and just publishing the placeholder if someone presses "Okay" without entering a string.